strings/src/main/res/values/strings.xml

```diff
@@ -982,4 +982,7 @@
     <string name="background_audio_help">This form records the audio from your device\'s microphone.\n\nYou can use the volume indicator to make sure the microphone is close enough to the sound you need to record.\n\nTo stop the recording please exit this form.\n\nFor more information please contact the person who asked you to collect data.</string>
 
     <string name="projects">Projects</string>
+    <string name="add_project">Add project</string>
```

Comment:
We should add comments for these strings really. Maybe we should add a "projects" group? @lognaturel did that for a bunch of other areas of the app.